choose_disk.yml

```diff
@@ -0,0 +1,41 @@
+---
+# Defines deployment design and assigns role to server groups
```

Comment:
What is the purpose of this playbook? Is this only for testing? If so, I don't think we need it included here.